index.js

```diff
@@ -1,8 +1,10 @@
 const express = require('express'),
       morgan = require('morgan'),
       bodyParser = require('body-parser'),
-      pug = require('pug'),
-      fs = require('fs');
+      fs = require('fs'),
+      pug = require('pug');
+
+var likeCount = JSON.parse(fs.readFileSync('likes.json')).likes;
 
 const userRoutes = require('./routes/users'),
       searchRoutes = require('./routes/search');
```

Comment:
http://expressjs.com/en/guide/routing.html : read the last title here: express.Router